tools/skbstat.py

```diff
@@ -0,0 +1,1015 @@
+#!/usr/bin/python
+# @lint-avoid-python-3-compatibility-imports
+#
+# skbstat Trace skb events(ingress/egress/consume/drop) of kernel.
+#         For Linux, uses BCC, eBPF. Embedded C.
+#
+# USAGE: skbstat [-h] [-i INTERVAL] [-s SADDR] [-d DADDR] [-N ADDR]
```

Comment:
Could you add a warning on potential performance impact on production system? Similar to tcptop.

Reply:
Sure. In fact, this tool will actually add some overhead. I'll add a warning.